src/pypromice/process/L0toL1.py

```diff
@@ -7,6 +7,8 @@
 import xarray as xr
 import re
 
+from pypromice.process.value_clipping import clip_values
```

Comment:
I really want to move functions out of aws.py, just like this. Should we just move them all out to individual files?